errors.yaml

```diff
@@ -0,0 +1,51 @@
+---
+errors:
+  - code: create_index
+    description: "Error relative to index creation, check out our guide on [index creation](link.com)"
+  - code: existing_index
+    description: "An index with this name already exists, check out our guide on [index creation](link.com)"
+  - code: invalid_index_uid
+    description: "The provided index formatting is wrong, check out our guide on [index creation](link.com)"
+  - code: open_index
+    description: "An error occured while trying to open the requested index, ..."
+  - code: invalid_state
+    description: "An error occuring when the database is in an invalid state. Deleting the database and reindexing should solve the problem."
+  - code: missing_primary_key
+    description: "All documents must be identified with a unique key. Meilisearch will try to infer the documetn field to use as primary key. If it can't, it has to be specified manually, and this error is returned. See <a href=\"https://docs.meilisearch.com/guides/main_concepts/documents.html#structure\">guide on primary keys</a>."
```

Comment:
```suggestion
    description: "All documents must be identified with a unique key. MeiliSearch will try to infer the document field to use it as a primary key. If it can't infer it, the primary key must be specified manually, and this error is returned. See <a href=\"https://docs.meilisearch.com/guides/main_concepts/documents.html#structure\">guide on primary keys</a>."
```